src/core/etl/tests/Flow/ETL/Tests/Benchmark/RowsBench.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+use Flow\ETL\Rows;
+use PhpBench\Attributes\Groups;
+use PhpBench\Attributes\Iterations;
+use PhpBench\Attributes\Revs;
+
+#[Iterations(5)]
+#[Groups(['rows'])]
```

Comment:
@norberttech Not sure if we should add a new group or re-use some existing ones.

Reply:
I think I would rename entry_factory into `building_blocks` and put NativeEntryfactory and Rows benchmarks under it, wdyt?

Reply:
Sounds good I guess, more general than just having `entry_factory` as one of the smallest groups.